### ISSUE TITLE
A11y_VS User Story 1350938_Account Options_Add new Account_Keyboard: When the user closes the 'Sign In' dialog with escape key, the focus disappears from the screen and on using 'tab' key, the focus lands on 'Learn More' link.

### ISSUE THREAD

**Issue by bbarik06** [dotnet/wpf#6664]
**GitHubTags:**#A11yMAS;#A11yTCS;#A11ySev2;#DesktopApp;#WCAG2.4.3;#Keyboard;#Win32;#LinkedBug;#BM-.NetCore(WPF)-Win32-July2021;#.NET Core;#Closed;

## Environment Details:
Application Name: Microsoft Visual Studio Enterprise 2022 (64-bit) - main
Windows Version: Win11

## Repro Steps:

1. Open Visual studio.
2. Tab till 'File' Menu item and hit enter on it.  
3. Tab till Account Setting and hit enter on it.
4. Tab till 'Add' button and hit enter on it. 
5. 'Sign In' dialog will open.
6. Close the dialog using keyboard and observe the focus order return.

## Actual:
When the user closes the 'Sign In' dialog with escape key, the focus disappears from the screen and on using 'tab' key, the focus lands on 'Learn More' link.

## Expected:
When the user closes the 'Sign In' dialog, the focus should return to the 'Add' button and should be visible to the users.

## User Impact:
Users with visual impairments will get wrong order of navigation

## Attachment:
![Presentation1](https://user-images.githubusercontent.com/98393966/172607053-d866de4f-855f-44fb-bac3-98679c627b97.gif)

**Comment by singhashish-wpf** [comment]
This is working fine in case of normal keyboard. The issue is occurring only in case of On-Screen Keyboard. It seems this is happening due to application focus switch between On-Screen Keyboard and the app.
